Tests/BootCamp.Chapter.Tests/ShopTests.cs

```diff
@@ -24,7 +24,7 @@ public void Add_New_Item_Adds_Item_To_Stock()
             var sword = new Item(ItemName, ItemPrice, 2);
             _shop.Add(sword);
 
-            _shop.Items.Should().Contain(sword);
+            _shop.Inventory.Items.Should().Contain(sword);
```

Comment:
Some slight changes to your ShopTests class so it verifies whether the inventory component contains the select items.